tests/rt.conf

```diff
@@ -350,47 +366,53 @@ RUN | rrfs_conus13km_hrrr_warm_debug                    | + hera cheyenne
+RUN | hrrr_control_restart_qr_dyn32_phy32               | + hera cheyenne            |          | hrrr_control_qr_dyn32_phy32
+RUN | rrfs_smoke_conus13km_phy32                        |                            | baseline |
+RUN | rrfs_smoke_conus13km_phy32_qr                     |                            | baseline |
+RUN | rrfs_smoke_conus13km_phy32_restart_mismatch       |                            | baseline | rrfs_smoke_conus13km_phy32
+RUN | rrfs_smoke_conus13km_phy32_restart_qr_mismatch    |                            |          | rrfs_smoke_conus13km_phy32_qr
 
-COMPILE | 43 | gnu | -DAPP=ATM -DCCPP_32BIT=ON | + hera cheyenne | fv3 |
+COMPILE | all_dyn64_phy32 | gnu | -DAPP=ATM -DCCPP_32BIT=ON | + hera cheyenne | fv3 |
 RUN | rap_control_dyn64_phy32                           | + hera cheyenne            | baseline |
 
-COMPILE | 44 | gnu | -DAPP=ATM -D32BIT=ON -DCCPP_32BIT=ON -DDEBUG=ON | + hera cheyenne | fv3 |
+COMPILE | all_dyn32_phy32_debug | gnu | -DAPP=ATM -D32BIT=ON -DCCPP_32BIT=ON -DDEBUG=ON | + hera cheyenne | fv3 |
 RUN | rap_control_debug_dyn32_phy32                     | + hera cheyenne            | baseline |
 RUN | hrrr_control_debug_dyn32_phy32                    | + hera cheyenne            | baseline |
 
-COMPILE | 45 | gnu | -DAPP=ATM -DCCPP_32BIT=ON -DDEBUG=ON | + hera cheyenne | fv3 |
+COMPILE | all_dyn64_phy32_debug | gnu | -DAPP=ATM -DCCPP_32BIT=ON -DDEBUG=ON | + hera cheyenne | fv3 |
 RUN | rap_control_dyn64_phy32_debug                     | + hera cheyenne            | baseline |
 
 ### S2S tests ###
-COMPILE | 46 | gnu | -DAPP=S2SWA -DCCPP_SUITES=FV3_GFS_v17_coupled_p8,FV3_GFS_cpld_rasmgshocnsstnoahmp_ugwp | + hera cheyenne | fv3 |
+COMPILE | p8_s2swa | gnu | -DAPP=S2SWA -DCCPP_SUITES=FV3_GFS_v17_coupled_p8,FV3_GFS_cpld_rasmgshocnsstnoahmp_ugwp | + hera cheyenne | fv3 |
 RUN | cpld_control_p8                                   | + hera cheyenne            | baseline |
 
-COMPILE | 47 | gnu | -DAPP=S2S -DCCPP_SUITES=FV3_GFS_v17_coupled_p8,FV3_GFS_cpld_rasmgshocnsstnoahmp_ugwp | + hera cheyenne | fv3 |
+COMPILE | p8_s2s | gnu | -DAPP=S2S -DCCPP_SUITES=FV3_GFS_v17_coupled_p8,FV3_GFS_cpld_rasmgshocnsstnoahmp_ugwp | + hera cheyenne | fv3 |
 RUN | cpld_control_nowave_noaero_p8                     | + hera cheyenne            | baseline |
 
-COMPILE | 48 | gnu | -DAPP=S2SWA -DDEBUG=ON -DCCPP_SUITES=FV3_GFS_v17_coupled_p8,FV3_GFS_cpld_rasmgshocnsstnoahmp_ugwp | + hera cheyenne | fv3 |
+COMPILE | p8_s2swa_debug | gnu | -DAPP=S2SWA -DDEBUG=ON -DCCPP_SUITES=FV3_GFS_v17_coupled_p8,FV3_GFS_cpld_rasmgshocnsstnoahmp_ugwp | + hera cheyenne | fv3 |
 RUN | cpld_debug_p8                                     | + hera cheyenne            | baseline |
 
 # Unstructured WW3
-COMPILE | 49 | gnu | -DAPP=S2SW -DCCPP_SUITES=FV3_GFS_v17_coupled_p8 -DPDLIB=ON | + hera cheyenne | fv3 |
+COMPILE | p8_pdlib | gnu | -DAPP=S2SW -DCCPP_SUITES=FV3_GFS_v17_coupled_p8 -DPDLIB=ON | + hera cheyenne | fv3 |
```

Comment:
These two ``pdlib`` cases should have the same name as the intel compiles.

Reply:
They match now.